wallet/cardano-wallet.js

```diff
@@ -255,24 +253,24 @@ const CardanoWallet = (secretOrMnemonic, CARDANOLITE_CONFIG) => {
     // eslint-disable-next-line no-undef
     end = CARDANOLITE_CONFIG.CARDANOLITE_ADDRESS_RECOVERY_GAP_LENGTH
   ) {
-    return (await deriveAddressesAndSecrets(begin, end)).map((item) => item.address)
+    return (await deriveAddressesWithHdNodes(begin, end)).map((item) => item.address)
   }
 
-  async function deriveAddressesAndSecrets(
+  async function deriveAddressesWithHdNodes(
     begin = 0,
     // eslint-disable-next-line no-undef
     end = CARDANOLITE_CONFIG.CARDANOLITE_ADDRESS_RECOVERY_GAP_LENGTH
   ) {
     const result = []
     for (let i = begin; i < end; i++) {
-      result.push(await getAddressAndSecret(0x80000001 + i))
+      result.push(await getAddressWithHdNode(0x80000001 + i))
```

Comment:
Interesting alternative might be
`await Promise.all(_.range(begin, end).map(async (i) => await getAddress()))`

or
```
range = _.range(begin + 0x8....1, end + 0x8...1)
await Promise.all(range.map(getAddressWith...()))
```

But I leave it open as to what is the most readable option